flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/Canceling.java

```diff
@@ -37,8 +37,15 @@
             ExecutionGraph executionGraph,
             ExecutionGraphHandler executionGraphHandler,
             OperatorCoordinatorHandler operatorCoordinatorHandler,
-            Logger logger) {
-        super(context, executionGraph, executionGraphHandler, operatorCoordinatorHandler, logger);
+            Logger logger,
```

Comment:
Just for the record: I verified that we do not collect failures while being already in cancelling stage for the `DefaultScheduler` (see [source](https://github.com/apache/flink/blob/35721033e18da6cb3e552de637942664b1ad879d/flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/Execution.java#L1111)). Hence, not implementing the archiving in this class is correct. 👍